staging/publishing/import-restrictions.yaml

```diff
@@ -1,245 +1,246 @@
-- baseImportPath: "./pkg/apis/core/"
-  allowedImports:
-  - k8s.io/apimachinery
-  - k8s.io/apiserver/pkg/util/feature
-  - k8s.io/component-base/featuregate/testing
-  - k8s.io/kubernetes/pkg/apis/core
-  - k8s.io/kubernetes/pkg/features
-  - k8s.io/kubernetes/pkg/fieldpath
-  - k8s.io/kubernetes/pkg/util
-  - k8s.io/api/core/v1
-  - k8s.io/utils/pointer
-  - k8s.io/utils/net
-  - k8s.io/klog
-
-  # the following are temporary and should go away. Think twice (or more) before adding anything here.
-  # Main goal: pkg/apis should be as self-contained as possible.
-  - k8s.io/kubernetes/pkg/apis/apps
-  - k8s.io/kubernetes/pkg/api/legacyscheme
-  - k8s.io/api/apps/v1
+- baseImportPath: './pkg/apis/core/'
```

Comment:
can you revert the formatting changes in this file to help see the actual changes being made?